README.md

```diff
@@ -27,3 +31,41 @@ The plugin for pretty printing [Qt][qt] types by [CLion][clion] debugger.
 ## Architectures tested on
 * x64
 
+## Requirements
+
+This plugin needs Debug information for Qt.
+
+If you installed Qt with Qt Online installer, ensure you have installed
+the `Qt Debug Information files`:
+
+![Checkbox for QtDebug Information files](images/Qt_Debug_Information_files_checkbox.png)
+
+If you are using Arch Linux, you need to install `qt6-base-debug` package.
+You can install manually (by specifying the url), or by enabling global repo. See [wiki](https://wiki.archlinux.org/title/Debugging/Getting_traces#Installing_debug_packages).
+```
+sudo pacman -U https://geo.mirror.pkgbuild.com/extra-debug/os/x86_64/qt6-base-debug-6.7.2-1-x86_64.pkg.tar.zst
+```
+
+## Troubleshooting
+
+### Qt Types are not pretty printed
+
+First, ensure you have satisfied [requirements](#Requirements).
+
+Now use gdb console to make a quick check if a specific version includes type metainfo by setting a breakpoint and running the command:
+
+```
+python print(gdb.parse_and_eval('*(&qtHookData)'))
+```
+
+There are problematic Qt versions, for which the debugger is unable to extract type metadata.
+
+For example, for Qt 6.4.2 you will get `<data variable, no debug info>` message:
+
+![No debug info in 6.4.2](images/bad_6.4.2_no_debug_info.png)
+
+Consider switching to another version of Qt.
+
+For the supported Qt version, you will get a message that looks like `{3, 7, 394754, 0, 0, 0, 22}`:
```

Comment:
It would be awesome if we have different instructions for LLDB and GDB. For example, for LLDB the output of the similar command would look like `01 06 06 00 00 00 00 00`